downstream/modules/platform/proc-provision-ocp-storage-amazon-s3.adoc

```diff
@@ -0,0 +1,49 @@
+[id="provision-ocp-storage-amazon-s3_{context}"]
+
+
+= Configuring object storage on Amazon S3
+
+Red Hat supports Amazon Simple Storage Service (S3) for {HubName}.
+You can configure it when deploying the `AutomationHub` custom resource (CR), or you can configure it for an existing instance.
+
+.Prerequisites
+
+* Create an Amazon S3 bucket to store the objects.
+* Note the name of the S3 bucket.
+
+.Procedure
+
+. Create a Kubernetes secret containing the AWS credentials and connection details, and the name of your Amazon S3 bucket.
+The following example creates a secret called `test-s3`:
++
+[source,yaml]
+----
+$ kubectl -n $PULP_NAMESPACE apply -f- <<EOF
```

Comment:
we should change it to:

```
$ oc -n $HUB_NAMESPACE
```

(kubectl would work, but oc is the Openshift alias for kubectl)